railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb

```diff
@@ -1,5 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# Do not halt callback chains when a callback returns false. This is a new Rails 5.0 default, 
-# so introduced as a config to ensure apps made with earlier versions of Rails aren't affected when upgrading.
+# Do not halt callback chains when a callback returns false. This is a new
+# Rails 5.0 default, so it is introduced as a config to ensure that apps made
+# on earlier versions of Rails are not affected when upgrading.
```

Comment:
This seems like quite a story, particularly given that the only people who see this (new 5.0 apps) really shouldn't care about it at all.

Have we previously been this verbose on similar config values?

(This is more of a general observation about the comment as it already stood, unrelated to your specific changes.)